smart-contracts/src/contract.rs

```diff
@@ -48,24 +59,9 @@ pub fn execute(
     msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
     match msg {
-        ExecuteMsg::AddFuel {
-            subdenom,
-            denom,
-            fuel_fee_recipient,
-            fuel_fee_rate,
-            ash_fee_recipient,
-            ash_fee_rate,
-        } => execute_add_fuel(
-            deps,
-            env,
-            info,
-            subdenom,
-            denom,
-            fuel_fee_recipient,
-            fuel_fee_rate,
-            ash_fee_recipient,
-            ash_fee_rate,
-        ),
+        ExecuteMsg::AddFuel { subdenom, denom } => {
+            execute_add_fuel(deps, env, info, subdenom, denom)
+        }
```

Comment:
if we do this, people won't be able to set themselves as fee recipients (neither for ash nor for fuel fees), is that what we want?

Reply:
Correct. We want a default fee in place that goes to the owner, or any other address if the owner likes to change it. The rationale here is that since people are not able to create a burn derivate for the same asset twice, i.e. each fuel asset is unique, a malicious user could front run the asset creation and set high fees only to himself.